src/internal/session.rs

```diff
@@ -1271,7 +1271,14 @@ mod tests {
     }
 
     #[test]
+    // The counter_mismatch test conforms to the following testing standards:
```

Comment:
We've added this line in a few places in wire-server, but I don't think it's helpful, and since it's not done anywhere else in this repo I would avoid introducing the noise.

Reply:
Sure, I'm removing it.